HomeFinder/HelpClass/HomeComparer.cs

```diff
@@ -0,0 +1,52 @@
+﻿using HomeFinder.Models;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace HomeFinder.HelpClass
+{
+    public class HomeComparer : IComparer<Home>
+    {
+        
+        //Class used with Sort to sort Homes in reverse order by GeneralGrade
+        public int Compare([AllowNull] Home x, [AllowNull] Home y)
```

Comment:
let's call x and y better

Reply:
I used this class for sorting so I figured it doesn't matter the names of these intermediate variables. To implement custom sorting I searched the Documentation. The variables in the example were called x and y(and x and y were dinosaurs objects)